e2e/specs/bridge/bridge-action-smoke.spec.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+import { loginToApp } from '../../viewHelper';
+import TabBarComponent from '../../pages/wallet/TabBarComponent';
+import {
+  loadFixture,
+  startFixtureServer,
+  stopFixtureServer,
+} from '../../fixtures/fixture-helper';
+import QuoteView from '../../pages/Bridge/QuoteView';
+import FixtureBuilder from '../../fixtures/fixture-builder';
+import FixtureServer from '../../fixtures/fixture-server';
+import WalletView from '../../pages/wallet/WalletView';
+import TestHelpers from '../../helpers';
+import { SmokeTrade } from '../../tags';
+import Assertions from '../../utils/Assertions';
+import Ganache from '../../../app/util/test/ganache';
+import WalletActionsBottomSheet from '../../pages/wallet/WalletActionsBottomSheet';
+import ActivitiesView from '../../pages/Transactions/ActivitiesView';
+import { ActivitiesViewSelectorsText } from '../../selectors/Transactions/ActivitiesView.selectors';
+import AddNewHdAccountComponent from '../../pages/wallet/MultiSrp/AddAccountToSrp/AddNewHdAccountComponent';
+import AccountListBottomSheet from '../../pages/wallet/AccountListBottomSheet.js';
+import AddAccountBottomSheet from '../../pages/wallet/AddAccountBottomSheet';
+import NetworkEducationModal from '../../pages/Network/NetworkEducationModal.js';
+import NetworkListModal from '../../pages/Network/NetworkListModal.js';
+import { getFixturesServerPort, getMockServerPort } from '../../fixtures/utils';
+import { startMockServer } from './bridge-mocks';
+import { stopMockServer } from '../../api-mocking/mock-server';
+import { localNodeOptions, testSpecificMock } from './costants';
+
+const fixtureServer = new FixtureServer();
+
+describe(SmokeTrade('Bridge from Actions'), () => {
```

Comment:
This is a minor nitpick, but I feel the spec name could be updated to `bridge-functionality.spec` file.

```suggestion
describe(SmokeTrade('Bridge functionality'), () => {
```